### PR TITLE
updated tester adding and searching to use new TestFlight API

### LINKED ISSUE #185: Method not allowed
We've had Boarding set up and working well for the last 2 months or so, however, all of the sudden it has started throwing Method Not Allowed errors when users attempt to sign up.

See attached.
screen shot 2017-11-15 at 4 17 11 pm

### PR DESCRIPTION
**Pull request by phaseOne**
Fixes / will fix #185

### CODE REVIEW

File: app/services/boarding_service.rb [hunk 1]
```diff
@@ -51,7 +51,7 @@ def add_tester(email, first_name, last_name)
         last_name: last_name,
         app: app
       )
-      if testing_is_live?
+      if true || testing_is_live? # TODO: remove true and test new train system
```
Comment (taquitos) [inline app/services/boarding_service.rb]: ☝️

**Comment by phaseOne** [comment]
@taquitos Testing with fastlane/fastlane#10937 branch now

**Comment by kdawgwilk** [comment]
Let me know if this is ready for me to give it a go and try it out for our team

**Comment by phaseOne** [comment]
Making one last fix. Should be ready in about 15min

**Comment by phaseOne** [comment]
good to go!

**Comment by kdawgwilk** [comment]
Tried to test this out and got the following error:

**Comment by phaseOne** [comment]
@kdawgwilk I received that error as well, but only if I didn't run `bundle update` and restart rails after switching branches. However, that might have been a coincidence.

**Comment by andydrizen** [comment]
I'm seeing an error in fastlane 2.66 owing to 

`if tester.kind_of?(Spaceship::Tunes::Tester::Internal)`

The error is:

> Something went wrong, please contact the application owner: uninitialized constant Spaceship::Tunes::Tester

**Comment by taquitos** [comment]
@andydrizen thanks, fixed in https://github.com/fastlane/boarding/pull/193

**Pull request merged**